controllers/survey.controller.js

```diff
@@ -1,11 +1,14 @@
 require('dotenv').config();
 const { pool } = require('../config');
 
+let isActive = 1;
+let deleteFlag = 0;
+let role_id = 3;
+const date = new Date();
 
-
-async function getSurveyData(user_id) {
+async function getSurveyData1(survey_id) {
```

Comment:
@sonalratho26 
Avoid using such names(getSurveyData1) that has numbers like 1,2,3